pkg/internal/reporter/reporter.go

```diff
@@ -19,15 +19,54 @@
 
 package reporter
 
-import "github.com/kiagnose/kubevirt-dpdk-checkup/pkg/internal/status"
+import (
+	"fmt"
+
+	"k8s.io/client-go/kubernetes"
+
+	kreporter "github.com/kiagnose/kiagnose/kiagnose/reporter"
+
+	"github.com/kiagnose/kubevirt-dpdk-checkup/pkg/internal/status"
+)
+
+const (
+	DropRate             = "dropRate"
+	TrafficGeneratorNode = "trafficGeneratorNode"
+	DPDKVMNode           = "DPDKVMNode"
```

Comment:
Please consider adding the `Key` suffix to these constants.

Reply:
Done